package.json

```diff
@@ -37,8 +43,8 @@
     "boot": "yarn prestart && yarn build",
     "start": "yarn prebuild && yarn storybook",
     "start:all": "yarn build && concurrently \"yarn docs dev\"  \"yarn storybook\" ",
-    "prebuild": "yarn pkg babel-plugin build",
-    "build": "lerna run build --ignore @chakra-ui/test-utils --no-private --stream",
+    "prebuild": "preconstruct dev && yarn pkg babel-plugin build && yarn pkg props-docs build",
```

Comment:
I added `yarn pkg props-docs build` here because we need to have that output built as part of the `build` command. I added `preconstruct dev` so that `props-docs` can reliably access the `propNames` export from `@chakra-ui/styled-system` when it runs.